modules/core/shared/src/main/scala/lucuma/core/math/skycalc/ImprovedSkyCalcMethods.scala

```diff
@@ -60,24 +60,24 @@ trait ImprovedSkyCalcMethods {
   }
 
   /**
-    * Return the airmass for the given altitude in degrees.
-    */
+   * Return the airmass for the given altitude in degrees.
+   */
   def getAirmass(alt: Double): Double = {
     val secz = secant_z(alt)
     if (secz >= 0.0)
       if (secz < 12.0) return true_airmass(secz)
-      else if (secz <= 99.0) return secz
+      else return secz
```

Comment:
Are the `return`s a hold-over from the Java code?  That is, should it be something like:

```scala
if (secz < 0.0) 0.0
else if (secz < 12.0) true_airmass(secz) else secz
```

Reply:
Yes, this is remnant from auto conversion from Java code. I'm removing these at least.